ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListQueriesExecutor.java

```diff
@@ -204,13 +204,16 @@ private ListQueriesExecutor() {
       final QueryId queryId = q.getId();
 
       // If the query has already been discovered, add to the ksqlQueryHostStatus mapping
+      // and the streams metadata task set
       if (allResults.containsKey(queryId)) {
         for (Map.Entry<KsqlHostInfoEntity, KsqlQueryStatus> entry :
             q.getKsqlHostQueryStatus().entrySet()) {
           allResults
               .get(queryId)
               .updateKsqlHostQueryStatus(entry.getKey(), entry.getValue());
         }
+        
+        allResults.get(queryId).updateTaskMetadata(q.getTasksMetadata());
```

Comment:
can we add a unit test case to `ListQueriesExecutorTest`?

Reply:
I extended the `show queries extended` unit test